### PR TITLE
chore (litmus-portal):  Added text to translations.yaml for components and pages

### PR DESCRIPTION
**Pull request by A4k9**
Signed-off-by: A4k9 <vansh.a4k9@gmail.com>

## Proposed changes
Added text to translations.yaml for components and pages.

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by arkajyotiMukherjee — DISMISSED** [review]
LGTM

**Review by S-ayanide — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 1]
```diff
@@ -1,43 +1,67 @@
...
 
+###########################################
+############    Components    #############
+###########################################
+
+sidebar:
+  title: Litmus
+
+header:
+  NotificationDropdown_NoNotifications: You don't have any new notification.
```
Comment (chandankumar4) [inline litmus-portal/frontend/public/locales/en/translation.yaml:35]:
```
header:
  notificationDropdown
    noNotifications: You don't have any new notification.
  profileDropdown
    editProfile: Edit Profile
    logOut: Log Out
    noProjects: You haven't created any projects yet.
```

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 2]
```diff
@@ -1,43 +1,67 @@
...
+  profileDropdownItems_EditProfile: Edit Profile
+  profileDropdownItems_LogOut: Log Out
+  profileDropdownItems_NoProjects: You haven't created any projects yet.
```
Comment (chandankumar4) [inline litmus-portal/frontend/public/locales/en/translation.yaml:38]:
```
header:
  notificationDropdown
    noNotifications: You don't have any new notification.
  profileDropdown
    editProfile: Edit Profile
    logOut: Log Out
    noProjects: You haven't created any projects yet.
```

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 3]
```diff
@@ -1,43 +1,67 @@
...
+
+quickactioncard:
+  quick_actions: Quick Actions
+  invite_team_member: Invite a team member
+  quick_survey: Take a quick survey
+  read_docs: Read The Litmus docs
```
Comment (chandankumar4) [inline litmus-portal/frontend/public/locales/en/translation.yaml:44]:
```
quickActionCard:
  quickActions: Quick Actions
  inviteTeamMember: Invite a team member
  quickSurvey: Take a quick survey
  readDocs: Read The Litmus docs
```

File: litmus-portal/frontend/public/locales/en/translation.yaml [hunk 4]
```diff
@@ -1,43 +1,67 @@
...
+
+workflowcard:
+  custom_workflow: Create your own workflow
```
Comment (chandankumar4) [inline litmus-portal/frontend/public/locales/en/translation.yaml:47]: Use camel case, just like above commnets

**Review by arkajyotiMukherjee — APPROVED** [review]
LGTM

**Review by chandankumar4 — DISMISSED** [review]
LGTM, Thanks

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**